components/pages/home/MemberCard.js

```diff
@@ -0,0 +1,48 @@
+import Image from "next/image";
+
+const MemberCard = ({ login, name, html_url, avatar_url }) => {
```

Comment:
Di export aja om tetep tapi jangan jadiin default

```suggestion
export const MemberCard = ({ login, name, html_url, avatar_url }) => {
```

Reply:
Yang di storybook sama ya, biar lebih enak buat coba coba. 

![image](https://user-images.githubusercontent.com/87353624/157194667-99e3c7dc-3876-441c-b59c-a42d11137829.png)

Reply:
Iya om samain aja gapapa